### PR TITLE
Added new options for ipmi-sensor to ITL

### PR DESCRIPTION
**Pull request by MarcusCaepio**
Regarding my PR at https://github.com/thomas-krenn/check_ipmi_sensor_v3/pull/42
I want to add the new options to the Icinga ITL.

### CODE REVIEW

File: itl/plugins-contrib.d/ipmi.conf [hunk 1]
```diff
@@ -74,6 +74,14 @@ object CheckCommand "ipmi-sensor" {
 			set_if = "$ipmi_show_fru$"
 			description = "Print the product serial number got by ipmi-fru"
 		}
+		"--assettag" = {
+			set_if = "$ipmi_show_assettag$"
+			description = "--fru is mandatory. Print the assettag if it is available in the IPMI FRU data."
```
Comment (Al2Klimov) [inline itl/plugins-contrib.d/ipmi.conf:79]: Why is here a full stop at the end, ...

File: itl/plugins-contrib.d/ipmi.conf [hunk 2]
```diff
@@ -74,6 +74,14 @@ object CheckCommand "ipmi-sensor" {
...
+		}
+		"--board" = {
+			set_if = "$ipmi_show_board$"
+			description = "--fru is mandatory. Print additional motherboard information if it is available in the IPMI FRU data"
```
Comment (Al2Klimov) [inline itl/plugins-contrib.d/ipmi.conf:83]: ... but here not.

File: doc/10-icinga-template-library.md [hunk 3]
```diff
@@ -2906,6 +2906,8 @@ ipmi_sensor_id                   | **Optional.** Include sensor matching ipmi_se
 ipmi_protocol_lan_version        | **Optional.** Change the protocol LAN version. Defaults to "LAN_2_0".
 ipmi_number_of_active_fans       | **Optional.** Number of fans that should be active. Otherwise a WARNING state is returned.
 ipmi_show_fru                    | **Optional.** Print the product serial number if it is available in the IPMI FRU data.
+ipmi_show_assettag               | **Optional.** --fru is mandatory. Print the assettag if it is available in the IPMI FRU data.
+ipmi_show_board                  | **Optional.** --fru is mandatory. Print additional motherboard informations if it is available in the IPMI FRU data.
```
Comment (Al2Klimov) [inline doc/10-icinga-template-library.md:2910]: Same here.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Consider putting "--fru is mandatory" at the end and in ( ).

Don't forget to use `git commit --amend`.

File: doc/10-icinga-template-library.md (hunk 3 above)
Comment (MarcusCaepio) [reply doc/10-icinga-template-library.md]: Corrected like you wanted.

**Review by Al2Klimov — APPROVED** [review]

**Review by julianbrost — APPROVED** [review]

**Pull request merged**